docker-compose-prod.yml

```diff
@@ -0,0 +1,86 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:12.13
+    environment:
+      PGDATA: /var/lib/postgresql/data/pgdata
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+    networks:
+      - panomicon_net
+    restart: unless-stopped
+    volumes:
+      - fusionauth_db_data:/var/lib/postgresql/data
+      - ./docker/fusionauth/sql:/docker-entrypoint-initdb.d
+
+  fusionauth:
+    image: fusionauth/fusionauth-app:1.42.0
+    depends_on:
+      db:
+        condition: service_healthy
+    environment:
+      DATABASE_URL: jdbc:postgresql://db:5432/fusionauth
+      DATABASE_ROOT_USERNAME: ${POSTGRES_USER}
+      DATABASE_ROOT_PASSWORD: ${POSTGRES_PASSWORD}
+      DATABASE_USERNAME: ${DATABASE_USERNAME}
+      DATABASE_PASSWORD: ${DATABASE_PASSWORD}
+      FUSIONAUTH_APP_MEMORY: ${FUSIONAUTH_APP_MEMORY}
+      FUSIONAUTH_APP_RUNTIME_MODE: development
+      FUSIONAUTH_APP_URL: http://fusionauth:9011
+    networks:
+      - panomicon_net
+    restart: unless-stopped
+    ports:
+      - 9011:9011
+    volumes:
+      - fusionauth_config:/usr/local/fusionauth/config
+
+  scalatra:
+    container_name: "scalatra"
+    build:
+      context: ./docker/scalatra
+      dockerfile: Dockerfile
+    depends_on:
+      - fusionauth
+    networks:
+      - panomicon_net
+    restart: unless-stopped
+    ports:
+      - 8888:8888
+    environment:
+      JVM_ARGS: ${SCALATRA_JVM_ARGS}
+      PORT: 8888
+      T_TS_URL: http://server-address:3030/Toxygates/query
```

Comment:
server-address is a placeholder value and should be changed when deploying, depending on how we deploy